### PR TITLE
complete_project

### PR DESCRIPTION
**Pull request by mtalhawahab099**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    `;
+    tabody.innerHTML += tablerow;
+}
+
+
+let Unique_key = () => {
+  if(localStorage.length > 0)
+  {
+    for (let i in localStorage){
+     let key2 = localStorage.key(i);
```
Comment (Zakarya525) [inline main.js:77]: Can we use the advanced array function here, **forEach** can be fitted here, what do you say?

File: main.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+      key++;
+    }
+    else{
+      key++;
+    }
+  }
+  return key;
+}
+}
+
```
Comment (Zakarya525) [inline main.js:88]: in both the **if** and **else** block you have _key++_, what do you mean here? if you have to increment the key in both cases then there is no need for **if** and **else** conditional blocks. Am I true?

File: main.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+// Submission Form Function
+let add = document.getElementById("add");
+ add.addEventListener("click", send_data = (event) => {  
+  // 👇️ if you are submitting a form (prevents page reload)
```
Comment (Zakarya525) [inline main.js:92]: "**send_data** is a function" do you have knowledge about JavaScript naming conventions🤔 if not please search about it ☺☺

File: main.js [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  {
+    themeIcon.classList.remove("fa-sun")
+    themeIcon.classList.add("fa-moon");
+  }
+  else{
+    themeIcon.classList.add("fa-sun")
+    themeIcon.classList.remove("fa-moon");
+  }
+
+}
```
Comment (Zakarya525) [inline main.js:165]: Good enough, have a great day😃

File: main.js (hunk 1 above)
Comment (mtalhawahab099) [reply main.js:77]: Actually, the key storage in the local storage is not in the form of array that's why we can't use advanced array function **"Foreach()"**

File: main.js (hunk 2 above)
Comment (mtalhawahab099) [reply main.js:88]: It was my mistake. I changed it in the local code and INSHAALLAH will be committed soon

File: main.js (hunk 3 above)
Comment (mtalhawahab099) [reply main.js:92]: INSHAALLAH I'll Search about it. **JAZAKALLAH**

File: main.js (hunk 4 above)
Comment (mtalhawahab099) [reply main.js:165]: **JAZAKALLAH** Ameer Saib for all the co-operation with me.

Comment (Zakarya525) [reply main.js:165]: Merged.

Comment (Zakarya525) [reply main.js:165]: because of the **Merged** now, do not initiate any pull requests against this branch.

Comment (Zakarya525) [reply main.js:165]: and it has been deleted now.

Comment (mtalhawahab099) [reply main.js:165]: Sahe da g, I'll commit the changes soon in the master branch

**Pull request merged**